integrations/purchase/nag-categories.js

```diff
@@ -33,371 +33,119 @@ import {
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Remodeling & Repair': {
-    purchaseType: PURCHASE_CATEGORY_STORE_HARDWARE,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Road Tax & Green Tax': {
-    purchaseType: PURCHASE_CATEGORY_OTHER_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Salary: null,
-  'Shared Expense': null,
-  'Sports & Leisure': {
-    purchaseType: PURCHASE_CATEGORY_RECREATIONAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Stock Trading': {
-    purchaseType: PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Student Grant': {
-    purchaseType: PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Student Loan': {
-    purchaseType: PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Supermarket: {
-    purchaseType: PURCHASE_CATEGORY_STORE_FOOD,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'TV license & Cable': {
-    purchaseType: PURCHASE_CATEGORY_INFORMATION_COMMUNICATION,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Taxi: {
-    purchaseType: PURCHASE_CATEGORY_TRANSPORT_ROAD,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Tobacco & Alcohol': {
-    purchaseType: PURCHASE_CATEGORY_ALCOHOL,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Transfer: {
-    purchaseType: PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Travel: {
-    purchaseType: PURCHASE_CATEGORY_OTHER_TRANSPORT_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Travel Insurance': {
-    purchaseType: PURCHASE_CATEGORY_INSURANCE,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Tuition: {
-    purchaseType: PURCHASE_CATEGORY_OTHER_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Unemployment Benefits': {
-    purchaseType: PURCHASE_CATEGORY_SOCIAL_PROTECTION,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Union & Unemployment Insurance': {
-    purchaseType: PURCHASE_CATEGORY_INSURANCE,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Unknown: null,
-  'Unpayed Tax': {
-    purchaseType: PURCHASE_CATEGORY_OTHER_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  Utilities: {
-    purchaseType: PURCHASE_CATEGORY_OTHER_ENERGY_HEATING_COOLING,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Vacation Activities': {
-    purchaseType: PURCHASE_CATEGORY_RECREATIONAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Vacation Home & Camping': {
-    purchaseType: PURCHASE_CATEGORY_RECREATIONAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Vacation Home Expenses': {
-    purchaseType: PURCHASE_CATEGORY_NON_DURABLE_HOUSEHOLD_GOODS,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
-  'Yield & Returns': {
-    purchaseType: PURCHASE_CATEGORY_FINANCIAL_SERVICES,
-    activityType: ACTIVITY_TYPE_PURCHASE,
-  },
+  'ATM & Checks': purchaseActivity(),
```

Comment:
`'ATM & Checks': null,` would be easier to read and understand that this is currently not mapped.